package.py

```diff
@@ -314,10 +314,15 @@ def open(self):
     def close(self, failed=False):
         self._zip.close()
         self._zip = None
+        if not os.exists(self._tmp_filename):
+            return
         if failed:
             os.unlink(self._tmp_filename)
         else:
-            os.replace(self._tmp_filename, self.filename)
+            try:
```

Comment:
This try/except block should not longer be needed with the check added above.

Reply:
Wow. Thats embarrassing 😮‍💨 ... Thanks for pointing that out though